client/components/mma/switch/SwitchOptions.tsx

```diff
@@ -84,10 +109,69 @@ const SwitchOptions = () => {
 						You're currently supporting the Guardian with a{' '}
 						{monthlyOrAnnual.toLowerCase()} contribution of{' '}
 						{mainPlan.currency}
-						{mainPlan.amount / 100}.
+						{currentAmount}.
 					</div>
 				</Card.Section>
 			</Card>
+			<div css={pageTopCss}>
```

Comment:
There's already  a `<Stack>` wrapping everything on the page so could that be adjusted to give you the spacing you need between the sections? (You might then want to use nested stacks to space elements within. eg. headings and cards.)

Reply:
I think the spacing should be different for mobile layout, but not sure if Stack spaces support that?

Reply:
Ah, okay. No, Stack doesn't currently support multiple spacing values for different breakpoints. I've got an idea how to do this, but need to put together a proposal and prototype 😅

I chatted to Helene about this whilst building the previous iteration and she was happy to leave the spacing the same for mobile and desktop, but we can leave it as is for now.